repaygithub/cactus
# CACTUS-329 :: Breadcrumb Updates

Tickets: https://repayonline.atlassian.net/browse/CACTUS-329

This has several breaking changes, so I'm merging it into a feature branch instead of master (we'll probably release v4 whenever we redesign the tables).

### Testing

1. Verify that the existing Breadcrumb stories look the same as the published version, as there should be no visible change, initially (the published version is https://repaygithub.github.io/cactus/stories/cactus-web/?path=/story/breadcrumb--basic-usage)
2. Look at the "Custom Item Elements" story, which demonstrates that you should be able to pass in a custom link component.
3. Verify that changing the theme color changes the breadcrumb colors as expected.
4. Verify that there is now a hover effect on the breadcrumb links that matches the style guide
5. Verify that the focus outline now matches the style guide, and that the border thickness theme setting has an effect.
